docs/en/concepts-and-designs/lal.md

```diff
@@ -10,7 +10,7 @@ activate specific LAL config files.
 
 ## Filter
 
-A filter is a group of [parser](#parser), [extractor](#extractor) and [sink](#sink). Users can use one or more filters
+A filter is a group of [parser](#parser), [extractor](#extractor), [SlowSql](#slowsql) and [sink](#sink). Users can use one or more filters
```

Comment:
I'd expect `extractor` to includes all operations that extract other things than log itself, can you move the `slowSql` into `extractor`, like `metrics` extractor? By doing this, we can reuse the logics like, service name, instance name, timestamp, layer, etc.

Reply:
OK, I will change it soon.